src/SFA.DAS.AssessorService.Web.Staff/Validators/Roatp/AddOrganisationProviderTypeViewModelValidator.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace SFA.DAS.AssessorService.Web.Staff.Validators.Roatp
+{
+    using FluentValidation;
+    using FluentValidation.Results;
+    using SFA.DAS.AssessorService.Api.Types.Models.Validation;
+    using SFA.DAS.AssessorService.Web.Staff.Resources;
+    using System.Collections.Generic;
+    using ViewModels.Roatp;
+
+    public class AddOrganisationProviderTypeViewModelValidator : AbstractValidator<AddOrganisationProviderTypeViewModel>
+    {
+        public AddOrganisationProviderTypeViewModelValidator()
+        {
+            RuleFor(vm => vm).Custom((vm, context) =>
+            {
+                var validationResult = IsValidProviderType(vm);
+                if (validationResult.IsValid) return;
+                foreach (var error in validationResult.Errors)
+                {
+                    context.AddFailure(error.Field, error.ErrorMessage);
+                }
+            });
+        }
+
+        private ValidationResponse IsValidProviderType(AddOrganisationProviderTypeViewModel viewModel)
+        {
+            var validationResult = new ValidationResponse
+            {
+                Errors = new List<ValidationErrorDetail>()
+            };
+            
+            var isValid = (viewModel.ProviderTypeId >= 1 && viewModel.ProviderTypeId <= 3);
```

Comment:
We'll do this some time as tech debt, but I'd like to isolate the rules for validation such as this one into its own service at some point.

Reply:
Agreed, I'd also like to put frequently used validation items such as UKPRN, Companies House Number into a Nuget package that can be shared with EPAO and other ESFA projects